src/main/java/de/uni_mannheim/informatik/dws/wdi/WDI_Project_T2/util/Utils.java

```diff
@@ -63,3 +93,40 @@ public static String preprocessTransmission(String transmission) {
         return transmission;
     }
 }
+
+class WhiteListSingleton {
```

Comment:
There should one be one class per file -> Move to a Utils/WhiteListSingleton file